content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/zebra.rst

```diff
@@ -0,0 +1,235 @@
+=========================
+Zebra label configuration
+=========================
+
+.. |ZPL| replace:: :abbr:`ZPL (Zebra Programming Language)`
+
+In Odoo, labels printed in the Zebra Programming Language (ZPL) file format are designed to fit a
+four-by-six inch label. To resize (or reformat) text to fit a variety of |ZPL| label sizes,
+:ref:`navigate to the ZPL label view <inventory/shipping_receiving/zpl-view>`, and alter the |ZPL|
+code.
+
+.. warning::
+   When customizing code in Odoo, please note that upgrading the database to newer versions may
+   break custom |ZPL| code. **Customers are responsible for maintaining their custom code**.
+
+Refer to the following sections for explanations, and example code, for frequently requested Zebra
+label customizations.
+
+- :ref:`adjust margins <inventory/shipping_receiving/margin>`
+- :ref:`enlarge/minimize barcodes <inventory/shipping_receiving/resize>`
+- :ref:`rotate elements <inventory/shipping_receiving/rotate>`
+
+.. _inventory/shipping_receiving/zpl-view:
+
+Navigate to ZPL label view
+==========================
+
+To begin customizing a Zebra label in Odoo, turn on :ref:`developer mode <developer-mode>`, and on
+the main Odoo dashboard, type `Reports`. From the search results that appear in the resulting pop-up
+window, choose :guilabel:`Settings/Technical/Reporting/Reports` to open the :guilabel:`Reports`
```

Comment:
I _believe_ there are spaces between the slash marks in the search box navigational address
```suggestion
window, choose :guilabel:`Settings / Technical / Reporting / Reports` to open the :guilabel:`Reports`
```